common/on-chip-calib.cpp

```diff
@@ -468,10 +518,9 @@ namespace rs2
                         break;
                 }
 
-                if (_sub->s->supports(RS2_OPTION_EMITTER_ENABLED))
-                    _sub->s->set_option(RS2_OPTION_EMITTER_ENABLED, 0.0f);
-                if (_sub->s->supports(RS2_OPTION_THERMAL_COMPENSATION))
-                    _sub->s->set_option(RS2_OPTION_THERMAL_COMPENSATION, 0.f);
+                // TODO - When implementing UV mapping - should remove from here and handle in process_flow()
+                set_laser_emitter_state( off_value );
```

Comment:
I see you removed the `supports` check, why?
Don't needed anymore?

Reply:
Maybe add it inside the helper funcs?

Reply:
It is checked in the helper functions. More "clean" this way, and less code repetitions